### ISSUE TITLE
[BUG]XEMM config sets parameters to none when no api key is entered

### ISSUE THREAD

**Issue by znerol19619** [CoinAlpha/hummingbot#1248]
**Describe the bug**
Xemm configuration sets some parameters to ``none`` value. Bug occurs when no api key is asked/entered during configuration. 

**Steps To Reproduce**
**Method 1**
1. Enable paper trade before configuration
2. Enter ``config`` command
3. Create XEMM strategy
4. Run ``list configs`` command

**Method 2** - This is for paper trade off
1. Create 2 XEMM strategy having the same configuration, where it won't ask for another api key on the 2nd strategy. 

_NOTE: There are times where it would still ask for an api key which are already saved on the ``hummingbot_conf folder``, which could be a bug aswell. ``none`` value won't show if it ask for an api key._

**Screenshots**
![papertradeon](https://user-images.githubusercontent.com/27669951/71512769-9538eb00-28d2-11ea-8076-6409b33f9213.png)
![Screenshot from 2019-12-27 18-16-36](https://user-images.githubusercontent.com/27669951/71513388-16917d00-28d5-11ea-9c62-a31379c44458.png)

**Release version**
```
                "branch": "development",
                "commit": "ae9aedcf9cc69254ebb8d3c2520a2d5ff88deed6",
                "date": "2019-12-26T23:59:48Z",

```

**Comment by znerol19619** [comment]
@veecos2

**Comment by Hashira-Lars** [comment]
Tested it with the latest dev image merged with PR #1245 the issue still persist. [NOT RESOLVED]

![image](https://user-images.githubusercontent.com/55869559/71568653-fb588480-2b03-11ea-8a21-1b3808571ae7.png)

**Comment by rennel-tabing** [comment]
Configuring each `None` parameters using the `config` command will make the XEMM work as a work around.

**Comment by Hashira-Lars** [comment]
@rennel-tabing thats correct , also entering your api keys will revert the none values to default values.

**Comment by Hashira-Lars** [comment]
Re-tested in master branch windows binary and issue still exist

![image](https://user-images.githubusercontent.com/55869559/72490021-e7758800-3850-11ea-854e-d5812e394509.png)

![image](https://user-images.githubusercontent.com/55869559/72490463-3e2f9180-3852-11ea-8f48-208056bcffa4.png)

**Comment by Hashira-Lars** [comment]
Re-tested with the latest master  `0.22.0` & latest dev `0.23.0` issue still persist

master
![image](https://user-images.githubusercontent.com/55869559/73224313-18847f80-41a4-11ea-9a05-0e0ae65e3f98.png)

dev
![image](https://user-images.githubusercontent.com/55869559/73224322-1fab8d80-41a4-11ea-8940-6b75d494c653.png)

**Comment by dennisocana** [comment]
Confirmed fix by @Clint-G13 in version 0.26.0
